### PR TITLE
Aug 680/create component

### PR DESCRIPTION
**Pull request by bakesaled**
Story: https://jira.inbcu.com/browse/AUG-680

A new  component that floats a pill with text above content:
image

### CODE REVIEW

File: src/badge.component.scss [hunk 1]
```diff
@@ -0,0 +1,39 @@
+@import "~suprematism-style-guide/dist/utils/_colors.scss";
+
+:host {
```
Comment (craigmichaelmartin) [inline src/badge.component.scss]: I think we want to have `pointer-events: none` here to pass mouse interactions through?

Comment (bakesaled) [inline src/badge.component.scss]: Yep, that's what's missing.  thx!

File: src/badge.component.scss [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+      -webkit-border-radius: 50%;
+      border-radius: 50%;
+      min-width: 1.5em;
+      display: flex;
+      flex-direction: row;
+      justify-content: center;
+      align-content: center;
+      line-height: 1.5;
+      pointer-events: none;
+    }
```
Comment (paulkahle) [inline src/badge.component.scss]: I would suggest a few tweaks to the style to ensure a circle for single digit numbers and a pill shape for larger numbers.
```
border-radius: 1em;
line-height: 2em;
padding: 0 .5em;
min-width: 2em;
```

Comment (bakesaled) [inline src/badge.component.scss]: @paulkahle With a big number, it appears more centered.  do you think this looks okay?
image
image

**Pull request merged**